src/components/project.js

```diff
@@ -0,0 +1,198 @@
+import { Header, HeaderLineBelow } from "../styles/Header"
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import Body from "../styles/Body.js"
+import { SB_ORANGE, SB_NAVY_RGBA } from "@colors"
+
+const CardBG = styled.div`
```

Comment:
I think the whole card should get `cursor: pointer` if it's all clickable, users will be confused about whether it's clickable otherwise.